components/support/ktx/src/main/java/mozilla/components/support/ktx/android/view/View.kt

```diff
@@ -38,13 +38,15 @@ fun View.isVisible(): Boolean {
 
 /**
  * Returns true if this view's visibility is set to View.GONE.
+ * @deprecated Use Android KTX instead.
  */
 fun View.isGone(): Boolean {
```

Comment:
I think we could at ` @Deprecated` [annotation](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-deprecated/index.html) on this function.